Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -33,7 +33,7 @@ struct TabLocationViewUX {
         theme.URLFontColor = UIColor.lightGray
         theme.textColor = UIColor(rgb: 0xf9f9fa)
         theme.highlightButtonColor = UIConstants.PrivateModePurple
-        theme.buttonTintColor = UIColor(rgb: 0xf9f9fa)
+        theme.buttonTintColor = UIColor(rgb: 0xADADb0)
```

Comment:
nit: caps

Reply:
a lot of these are all over the place right now. My plan is to fix this in the great color fixup